internal/cmd/mariadb/credentials/create/create.go

```diff
@@ -0,0 +1,119 @@
+package create
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/mariadb/client"
+	mariadbUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/mariadb/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/mariadb"
+)
+
+const (
+	instanceIdFlag   = "instance-id"
+	hidePasswordFlag = "hide-password"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	InstanceId   string
+	HidePassword bool
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates credentials for an MariaDB instance",
```

Comment:
```suggestion
		Short: "Creates credentials for a MariaDB instance",
```

Same here for all descriptions where this occurs